manifests/master.userdata_secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: master-user-data-managed
```

Comment:
(Not to totally derail this but we *do* have an opportunity now to call this `controlplane-user-data-managed` or so; the [installer config](https://github.com/openshift/installer/blob/master/docs/user/customization.md) uses the terms "controlPlane" and "compute" and not master/worker...just noting this)

Reply:
awesome :star_struck: